### ISSUE TITLE
Run without prompt

### ISSUE THREAD

**Issue by lfaoro** [mail-in-a-box/mailinabox#702]
Is there a way for us to run the start.sh script silently specifying the settings on the command line and avoid getting domain name prompt?

**Comment by JoshData** [comment]
Yep, see https://github.com/mail-in-a-box/mailinabox/blob/master/Vagrantfile#L21.

**Comment by JoshData** [comment]
Also #737.

**Comment by bilogic** [comment]
![Image](https://github.com/user-attachments/assets/78925078-306a-4d5d-83e8-e5c22e530dd8)

I ran `sudo PRIMARY_HOSTNAME=auto mailinabox` but still getting the above in `v71a`

**Comment by kiekerjan** [comment]
> I ran `sudo PRIMARY_HOSTNAME=auto mailinabox` but still getting the above in `v71a`

You should also set `NONINTERACTIVE=1`, `PUBLIC_IP=auto` and `PUBLIC_IPV6=auto`
